feature-libs/product/configurators/common/styles/_configurator-product-title.scss

```diff
@@ -24,21 +24,21 @@
       -webkit-transition: all 0.7s ease-in-out;
 
       &.open {
-        /* 
-          A larger value than the details need, this will be used to animate the closing/opening. 
+        /*
+          A larger value than the details need, this will be used to animate the closing/opening.
           If the value is to large the closing will "delay" as the max-height will be reduced from the specified value down to 0.
         */
-        max-height: 400px;
+        max-border-left: 400px;
```

Comment:
is this correct?

Reply:
**max-height** => **max-block-size**
**max-border-left**: => I guess: **border-inline-start-width**

Reply:
Done!